src/acl/Node.h

```diff
@@ -30,14 +31,19 @@ class Node
 
     static void ParseAclLine(ConfigParser &parser, Acl::Node **head);
     static void Initialize();
+
+    /// A configured ACL with a given name or nil.
+    static Acl::Node *FindByName(const SBuf &);
+    /// \copydoc FindByName()
+    /// \deprecated Use to avoid performance regressions; remove with the last caller.
     static Acl::Node *FindByName(const char *name);
```

Comment:
This legacy function has one remaining caller: Acl::InnerNode::lineParse(). It will disappear as we upgrade squid.conf parsing code to use SBuf, but I think we should keep it for now because the remaining caller may be called frequently in some large configurations, and the corresponding to-SBuf conversion overheads (that would be added if this PR removes this function) may be noticed.

I adjusted FindByName() implementation so that the two FindByName() functions declared above do not duplicate each other code.